backend/sirius-web-collaborative-forms-api/src/main/java/org/eclipse/sirius/web/collaborative/forms/api/dto/UpdateWidgetFocusSuccessPayload.java

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2020 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.collaborative.forms.api.dto;
+
+import java.text.MessageFormat;
+import java.util.Objects;
+
+import org.eclipse.sirius.web.annotations.graphql.GraphQLField;
+import org.eclipse.sirius.web.annotations.graphql.GraphQLNonNull;
+import org.eclipse.sirius.web.annotations.graphql.GraphQLObjectType;
+import org.eclipse.sirius.web.services.api.dto.IPayload;
+
+/**
+ * The payload of the "Widget Focus" mutation returned on success.
+ *
+ * @author smonnier
+ */
+@GraphQLObjectType
+public final class UpdateWidgetFocusSuccessPayload implements IPayload {
+    private final UpdateWidgetFocusInput widget;
+
+    public UpdateWidgetFocusSuccessPayload(UpdateWidgetFocusInput widget) {
+        this.widget = Objects.requireNonNull(widget);
+    }
+
+    @GraphQLField
+    @GraphQLNonNull
+    public UpdateWidgetFocusInput getUpdateWidgetFocusInput() {
+        return this.widget;
+    }
+
+    @Override
+    public String toString() {
+        String pattern = "{0} '{'widget focused: '{'project_id: {1}, representation_id: {2}, widget_id: {3}'}''}'"; //$NON-NLS-1$
```

Comment:
We normally use `fooId` instead of `foo_id` here. See for example `org.eclipse.sirius.web.collaborative.api.dto.CreateRepresentationInput.toString()`.

Reply:
Done